src/style.css

```diff
@@ -1 +1,7 @@
-/* styles go here */
+.lineSpacing div:not(:last-child) {
```

Comment:
Checklist point 6. 
Instead of wrapping all inputs and divs around them in another div, maybe try to style each of those divs around the inputs.